### PR TITLE
Component/SearchButton Implementation

### PR DESCRIPTION
**Pull request by pedram-am**
The Search Button component has been implemented, which has been extracted from the existing Search component for improved modularity and reusability.

### CODE REVIEW

File: src/components/SearchButton/SearchButton.test.jsx [hunk 1]
```diff
@@ -3,13 +3,9 @@ import { describe, it, expect } from "vitest";
 import { render, screen } from "@testing-library/react";
 
 describe("Search Button", () => {
-  it("should have a test, please write one", () => {
-    return true;
+  it("renders a search button", () => {
+    render(<SearchButton />);
+    const searchButton = screen.getByRole("button");
+    expect(searchButton).toBeInTheDocument();
   });
```
Comment (HadikaMalik) [inline src/components/SearchButton/SearchButton.test.jsx:10]: Good job!

File: src/components/Search/Search.jsx [hunk 2]
```diff
@@ -15,7 +17,7 @@ const Search = () => (
         className="search__input"
         placeholder="Customer name"
       />
-      <button className="button search__button">Search</button>
+      <SearchButton />
     </form>
   </section>
 );
```
Comment (HadikaMalik) [inline src/components/Search/Search.jsx:23]: Looks good, well done!

**Review by HadikaMalik — APPROVED** [review]
Your code makes sense and looks good.

**Comment by HadikaMalik** [comment]
When I merged your work, it was giving me an error. 
in your serach button implementation you wrote 
import SearchButton from "./SearchButton/SearchButton";
I guess you missed one . by accident. 
it is supposed to be like this:
import SearchButton from "../SearchButton/SearchButton";

File: src/components/Search/Search.jsx [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import React from "react";
+import SearchButton from "./SearchButton/SearchButton";
```
Comment (eliza-an) [inline src/components/Search/Search.jsx:2]: this should be ../SeachButton/search button since you are importing it from outside of the current folder

**Pull request merged**